examples/i3s/deck.gl/i3s-3d-layer/i3s-3d-layer.js

```diff
@@ -4,12 +4,12 @@ import {Vector3} from 'math.gl';
 import {COORDINATE_SYSTEM, CompositeLayer} from '@deck.gl/core';
 import {SimpleMeshLayer} from '@deck.gl/mesh-layers';
 
-import {I3STileset} from '@loaders.gl/i3s';
+import {I3STileset3D} from '@loaders.gl/i3s';
```

Comment:
Q: Is the plan to merge this tileset class with the other one?

We can e.g. create a `@loaders.gl/category-3d-tiles` if we need a common place.

Reply:
Does `3d-tiles` cover both cesium `3d-tiles` and `i3s`?

Reply:
> Does `3d-tiles` cover both cesium 3d-tiles and i3s?

Assuming you mean the `@loaders.gl/3d-tiles` npm module, I am open to making a change here, and have it include multiple formats.
- A strong reason to join them would be to avoid inefficiency / overhead of having so many npm modules). 
- A reason to keep them separate could be to make it easier for the various partners that help us maintain the different standards to make changes to their module without being affected by other modules. But since we are sharing traversal code anyway I am not sure this really helps.

If joining the modules, some considerations would be:
- We do want a clear distinction between code that implements a certain tiled format (subdirectories for `./3d-tiles/`, `./i3s/`, `./potree/` etc). This would contain the various file format loaders, but also format specific things like styling support etc.
- Clearly documented `./common/` code that is shared between formats (traversal/caching/request scheduling etc).

Of course we should also consider the opinions of contributors, e.g. if @loshjawrence would have concerns about a single module setup covering all tiled formats.